revelations/perspire-demo/33bea1e3-9aac-4181-a767-3427f58375f7.json

```diff
@@ -0,0 +1 @@
aws.com/developtest"}}},{"actions":[{"typeName":"Property","id":["8aa79caf-a9c1-454f-82a2-0120c8108edc",null,null,"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest?id=33bea1e3-9aac-4181-a767-3427f58375f7"],"initialState":{"owner":["33bea1e3-9aac-4181-a767-3427f58375f7","properties",null,"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest?id=33bea1e3-9aac-4181-a767-3427f58375f7"],"name":"value","value":{"typeName":"Literal","value":null}},"type":"CREATED","commandId":"8aa79caf-a9c1-454f-82a2-0120c8108edc","partitions":{"33bea1e3-9aac-4181-a767-3427f58375f7":{"eventId":null,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}},{"id":["040da48c-9fc1-4219-9a30-d8af82d066fa",null,null,"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest?id=33bea1e3-9aac-4181-a767-3427f58375f7"],"typeName":"Property","type":"REMOVED_FROM","removes":{"value":null},"commandId":"a09b050e-1f10-42fa-bf3f-0fd130869fb0","partitions":{"33bea1e3-9aac-4181-a767-3427f58375f7":{"eventId":5,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}}],"type":"TRANSACTED","version":"0.1","timeStamp":1534252344941,"commandId":"9a8d8408-54d8-4043-8002-b6c002b43081","partitions":{"33bea1e3-9aac-4181-a767-3427f58375f7":{"eventId":5,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}},{"actions":[{"id":["8aa79caf-a9c1-454f-82a2-0120c8108edc",null,null,"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest?id=33bea1e3-9aac-4181-a767-3427f58375f7"],"typeName":"Property","type":"FIELDS_SET","sets":{"value":{"typeName":"Literal","value":0}},"commandId":"1be1d7cd-32fd-4d29-871c-63be7b6a4053","partitions":{"33bea1e3-9aac-4181-a767-3427f58375f7":{"eventId":6,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}}],"type":"TRANSACTED","version":"0.1","timeStamp":1534252347330,"commandId":"06ae0510-8e6e-4001-92d2-a891fc2163e0","partitions":{"33bea1e3-9aac-4181-a767-3427f58375f7":{"eventId":6,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}},{"actions":[{"id":["8aa79caf-a9c1-454f-82a2-0120c8108edc",null,null,"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest?id=33bea1e3-9aac-4181-a767-3427f58375f7"],"typeName":"Property","type":"FIELDS_SET","sets":{"value":{"typeName":"Literal","value":1}},"commandId":"53e482fe-370a-4608-bc9c-057d9915fe46","partitions":{"33bea1e3-9aac-4181-a767-3427f58375f7":{"eventId":7,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}}],"type":"TRANSACTED","version":"0.1","timeStamp":1534252354465,"commandId":"de0c8164-db78-49ee-b04c-6adeee2ed79e","partitions":{"33bea1e3-9aac-4181-a767-3427f58375f7":{"eventId":7,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}}]}
```

Comment:
Need to string-replace the `valaa-aws://j44low.../developtest` with `valaa-local:`
Please do not commit the string "valaa-aws" anywhere in the @valos/vault repository. 
@valos/vault does not know about valaa-aws.